app/views/report/_form_sort.html.haml

```diff
@@ -142,13 +142,12 @@
               - if ci[:numeric]
                 #form-group
                 - field_name = "calc_id_#{f_idx}"
-                - col = field_to_col(f.last)
-                - selected = @edit[:new][:col_options][col] && @edit[:new][:col_options][col][:grouping] ? @edit[:new][:col_options][col][:grouping] : []
+                - opts = [[_("Minimum"),"Minimum"],[_("Average"),"Average"], [_("Maximum"), "Maximum"], [_("Total"),"Total"]]
                 = select_tag(field_name,
-                    options_for_select(MiqReport::GROUPINGS.map(&:reverse), selected),
```

Comment:
@Maria-Cordero-ibm i think you could have done something like `MiqReport::GROUPINGS.map(&:reverse).map { |key, value| [_(key), value] }` to fix this.

Reply:
That's clever, I changed the solution to this

Reply:
I'm not sure if this way the data gets indexed properly into the translation DB, maybe we should call the `_` where [the constant is declared](https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_report.rb#L55).

Reply:
We'd probably need `N_()` in the model class (since it's a constant) and `_()` here.

Reply:
I updated the core repo to reflect the changes and made a PR [#20537](https://github.com/ManageIQ/manageiq/pull/20537).